UP.sh

```diff
@@ -24,17 +24,27 @@ then
 fi
 
 File="docker-compose.yml"
-TestOveride=""
+TestOverride=""
 if [[ -n $Dev ]]
 then
     File="docker-compose.dev.yml"
 fi
-if [[ -n $Test ]]
+if [ "$PLATFORM" = "podman" ]
 then
-    TestOveride="-c ./test/setup/ci-test.overide.yml"
+   if [[ -n $Test ]]
+   then
+     TestOverride="-f ./test/setup/ci-test.overide.yml"
+   fi
+   podman compose -f $File -f docker-compose.override.yml $TestOverride -p $STACKNAME up -d
+	# systemctl --user start pod-${STACKNAME}.service
+else
+   nohup node ab_system_monitor.js &> /dev/null &
```

Comment:
do we not run `ab_system_monitor.js` in a `podman` case?

Reply:
My preference would be to not to run it at all. It's meant to be used with the `bot_manager` service which is not really a part of our plans. We're not using it anywhere that I know of, and not sure it works the way we would want it to. I definitely don't want to be using bot manager on the Core-Os, without a use case and confidence that it works correctly.
Also don't see podman being used in many other places, so for now I don't think we need to run it.

Reply:
That's fine.

If we no longer want the features `bot_manager` was supposed to provide for us, we should discuss taking it off the plate of our installed resources.  MIght be a good discussion for another time.